xray/core/plotting.py

```diff
@@ -0,0 +1,16 @@
+def _plot_dataarray(darray, *args, **kwargs):
+    """
+    Plot a DataArray
+    """
+    import matplotlib.pyplot as plt
+
```

Comment:
We have been considering that. `pandas` plotting does much more than we're planning to put here. The goal here is to use the metadata to make simple plots.